### ISSUE TITLE
fatal error for 0.9.4

### ISSUE THREAD

**Issue by secarica** [MartijnBraam/pyatem-win#3]
While trying to run version 0.9.4 on Windows 10 64bit, I get a fatal error – see the attached screenshot. Actual screenshot is from .msi installed version, but I get a similar error thing trying to run the .zip version. I also installed MinGW 'just in case', but no difference.

Few months ago I also tried 0.9.2 which gave fatal error – don't know if the same, as I didn't recorded any log.

??

![openswitcher_0 9 4_windows_10](https://github.com/MartijnBraam/pyatem-win/assets/20367333/c9542d70-a8bc-452c-b0f8-5a14604732a2)

**Comment by MartijnBraam** [comment]
Yep, the windows builds are horribly broken somewhere deep inside the mingw gtk3 stuff somewhere and I can't figure out why it doesn't work.

That AssertionError is deep inside GTK3 internals.

None of this software was originally intended to run on windows and I guess the WSL stuff is probably a better way to run it.
